project-codeflare/codeflare-sdk
# Status, Details, and List rayclusters CLI Functions

# Issue link
Closes #240 

# What changes have been made
- Added `list rayclusters` `details raycluster` and `status raycluster` functions in the CLI
- Added `list_clusters_all_namespaces` to list all RayClusters in all the namespaces in a Kubernetes Cluster

# Verification steps
After building SDK, see list of commands using `codeflare`

- `list rayclusters` for listing, optionally specifying namespace or `--all`
- `status raycluster` specifying name as an argument and namespace as an option
- `details raycluster` specifying name as an argument and namespace as an option

There are also unit tests for these functions to be run
## Checks
- Testing Strategy
